### ISSUE TITLE
Investigate into Wally usage tracking

### ISSUE THREAD

**Issue by jakemorr** [bcgov/Sustainment-Team#18]
**Describe the task**
an investigation on if wally has any type of usage data we can give access to or get for the service owner.

Any data we can provide on how many active users wally has, how often they are using the application on average and how many reports are generated over a given time period would be valuable. any other usage related data could be useful too if found. 

if there is nothing tracking usage data right now, we will want to investigate possible ways to collect such data in the simplest way. 

**Purpose**
 the service owner is currently gathering information for the ministry to make a decision on which Water Allocation Tool should be rolled out province wide

**Acceptance Criteria**
- [ ] investigate if any tracking solution is active
- [ ] provide any data that is available

**Additional context**
- if there is no tracking solution available, create a new ticket.

**Comment by Keegnan** [comment]
Could be a good first issue.

**Comment by Keegnan** [comment]
Ideally to be ready by Friday, July 26th (if possible).

**Comment by sggerard** [comment]
WALLY definitely had Motomo installed for tracking analytics. However only currently the motomo db pod is running and the motomo proxy and analytics pods don't appear to have run in a long time. Will investigate if we can get these pods running again. https://github.com/BCDevOps/matomo-openshift/tree/master

WALLY also appears to use https://github.com/stephenhillier/starlette_exporter and can export metrics that could be consumed by a prometheus deployment on the /metrics endpoint. This doesn't appear to be used currently, and I haven't been able to check the endpoint yet as I do not have permission. Relevant code: https://github.com/bcgov/WALLY/blob/eae559720c25642a83e2a51ba99c8b5276d2cfad/backend/main.py#L13

**Comment by sggerard** [comment]
Motomo appears to be broken as it's pulling the image from the wrong URL.
See relevant PR with the correct URL https://github.com/BCDevOps/matomo-openshift/commit/6b826fa514ca91266d946fae53738db7bce9672d

**Comment by sggerard** [comment]
The frontend is trying to call motomo, so likely just need to get the pod running and motomo will be tracking analytics again.

By port forwarding the api I was able to hit the /metrics endpoint. (oc port-forward wally-production-api-41-hp7bd 8000:8000).
I needed to do this for both pods. It looks like these pods have been running since Feb 27th.

[Metrics.txt](https://github.com/user-attachments/files/16416840/Metrics.txt)

[metrics2.txt](https://github.com/user-attachments/files/16416905/metrics2.txt)
